test/typed_buffers_test.dart

```diff
@@ -289,38 +293,38 @@ void testIntBuffer(int bits, int min, int max,
 
 const List doubleSamples = const [
   0.0,
-  5e-324,                    // Minimal denormal value.
-  2.225073858507201e-308,    // Maximal denormal value.
-  2.2250738585072014e-308,   // Minimal normal value.
-  0.9999999999999999,        // Maximum value < 1.
+  5e-324, // Minimal denormal value.
```

Comment:
Please add spaces after the slashes so the comments line up again.

Reply:
ack